core/vm/evm.go

```diff
@@ -51,8 +53,20 @@ func (evm *EVM) precompile(addr common.Address) (PrecompiledContract, bool) {
 	default:
 		precompiles = PrecompiledContractsHomestead
 	}
+
+	// Check the existing precompiles first
```

Comment:
This is important to behavior to test -- let's add a test that overrides an existing precompile address with a contract that panics and test that it does not override it.

Reply:
Fixed https://github.com/Kava-Labs/go-ethereum/pull/10/commits/d85296e76ae9da578bb199d2a82207157263623b

In `TestEvmIsPrecompileMethod` test, I'm testing only `stateful precompile` part, but we can consider adding tests for `native stateless precompile` as well if you think it make sense.